wp-includes/service-workers.php

```diff
@@ -334,3 +334,25 @@ function wp_service_worker_fragment_redirect_old_slug_to_new_url( $link ) {
 	}
 	return $link;
 }
+
+/**
+ * Get service worker error messages.
+ *
+ * @return array Array of error messages: default, comment.
+ */
+function wp_service_worker_get_error_messages() {
```

Comment:
As mentioned above too, I'm wondering if this method is necessary as a global method.